index.js

```diff
@@ -20,3 +20,30 @@ module.exports = fn => {
 
 	return ret;
 };
+
+const log = (fn, promise) => {
+	console.log(`Promise from ${fn.displayName} resolved in ${promise.time} ms`);
```

Comment:
Tested this on both Node 4 and 6, but on my system it's the other way around. Is there a relationship between `name` and `displayName`? Because `displayName` is just a regular property on the function `f`, no?

```js
console.log(f.name);
//=> undefined
console.log(f.displayName);
//=> 'foo'
```